descope/management/access_key.py

```diff
@@ -123,7 +123,7 @@ def update(
         self,
         id: str,
         name: str,
-        description: str,
+        description: Optional[str] = None,
```

Comment:
in this case i'd go with the default value `description: str = ""` since it will go on the payload anyway